examples/annotation-canvas-crosshair/index.js

```diff
@@ -0,0 +1,50 @@
+const data = [{ x: 450, y: 450 }];
+
+const container = document.querySelector('d3fc-canvas');
+
+const xScale = d3.scaleLinear().domain([0, 1]);
+
+const yScale = d3.scaleLinear().domain([0, 1]);
+
+const format = d3.format('.2f');
+
+const crosshair = fc
+    .annotationCanvasCrosshair()
+    .xScale(xScale)
+    .yScale(yScale)
+    .xLabel(d => format(xScale.invert(d.x)))
+    .yLabel(d => format(yScale.invert(d.y)))
+    .xDecorate(context => {
+        context.strokeStyle = 'rgba(204, 0, 0, 0.25)';
+    })
+    .yDecorate(context => {
+        context.strokeStyle = 'rgba(204, 0, 0, 0.25)';
+    })
+    .decorate(context => {
+        context.strokeStyle = 'rgba(204, 0, 0, 0.25)';
+        context.fillStyle = 'rgba(0, 0, 0, 0)';
+        context.scale(95, 95);
+    });
+
+const canvas = container.querySelector('canvas');
+
+d3.select(container)
+    .on('draw', () => {
+        crosshair(data);
+        d3.select(canvas).on('mousemove', () => {
```

Comment:
The canvas should be the same size as the container so there's no need to reference it. I think that would allow you to move this handler to be a peer of 'draw', 'measure'.
```suggestion
        d3.select(contain).on('mousemove', () => {
```